pom.xml

```
@@ -101,11 +101,21 @@ THE SOFTWARE.
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
+            <version>2.1.17</version>
```

```suggestion
            <version>2.3.5</version>
```

This is fairly old otherwise.